easybuild/tools/job/gc3pie.py

```diff
@@ -96,9 +93,19 @@ def init(self):
 
         Start a new list of submitted jobs.
         """
-        self.output_dir = os.path.join(os.getcwd(), 'easybuild-gc3pie-jobs')
+        cfgfile = build_option('job_backend_config')
+        if cfgfile:
+            # FIXME: is there a better way?
```

Comment:
any better way to handle this?

setting `$GC3PIE_CONF` doesn't work anymore at this point